webapp/src/components/sidebar_left/sidebar_left.jsx

```diff
@@ -0,0 +1,60 @@
+
+
+import React, {useEffect} from 'react';
+
+import PropTypes from 'prop-types';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import {Tooltip, OverlayTrigger} from 'react-bootstrap';
+
+
+const SidebarLeft = ({ id, installs, actions, showRHS, theme }) => {
+    useEffect(() => {
+        actions.getCloudUserData(id);
+    }, [id, actions]);
+    
+    const style = getStyle(theme);
+
+    return (
+        <div className='CloudPlugin__SidebarLeft'>
+            <OverlayTrigger
+                key='githubYourPrsLink'
```

Comment:
Probably should be a different key? Guessing it doesn't matter much, but would be good to have updated.

Reply:
Can you tell where I borrowed the styling from?? Removed - this isn't actually in a list so no need for a key like the github plugin would have